### DISCUSSION TITLE
Option to change a tab label's background color

### DISCUSSION THREAD

**Discussion by myz-dev** [zellij-org/zellij#2360] [Q&A]
Hi,
when I have multiple tabs open, I sometimes wish I could color their labels differently (the option to rename them is already very useful!).
Is this possible at the moment?
If not, could someone give me a pointer to where in the code I would look for trying to implement something like this?

**Comment by erictheswift** [comment]
Or have accent color to be selected dynamically (e.g. based on title)
Had this kind of setup in tmux on hooks, figuring out how to color code things dynamically in zellij
